src/main/resources/runners/sigtest/pom.xml

```diff
@@ -127,22 +128,38 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.netbeans.tools</groupId>
+                <!--                <groupId>org.netbeans.tools</groupId>-->
+                <groupId>jakarta.tck</groupId>
                 <artifactId>sigtest-maven-plugin</artifactId>
+                <version>2.2</version>
                 <executions>
-                  <execution>
-                    <id>default-cli</id>
-                    <phase>integration-test</phase>
-                    <goals>
-                      <goal>check</goal>
-                    </goals>
-                  </execution>
+                    <execution>
+                        <id>default-cli</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>check</goal>
+                            <!--                            <goal>generate</goal>-->
+                        </goals>
+                    </execution>
                 </executions>
                 <configuration>
-                  <action>strictcheck</action>
-                  <failOnError>true</failOnError>
-                  <packages>jakarta.batch.**</packages>
-                  <classes>${project.build.directory}/dependency</classes>
+                    <action>strictcheck</action>
+                    <failOnError>true</failOnError>
+                    <packages>jakarta.batch.**</packages>
+                    <classes>${project.build.directory}/dependency</classes>
```

Comment:
This will pass if you add `<release>11</release>` to let it know what the target release version is.

Reply:
Cool! I'll try to verify this.